packages/@aws-cdk-testing/framework-integ/test/aws-lambda/test/integ.vpc-lambda.ts

```diff
@@ -14,4 +15,14 @@ new lambda.Function(stack, 'MyLambda', {
   vpc,
 });
 
-app.synth();
+new lambda.Function(stack, 'IPv6EnabledLambda', {
```

Comment:
I tried the existing PR and its integration tests were failing, here's my fork with changes to integration tests and almost similar changes for the feature implementation that works: https://github.com/Vandita2020/aws-cdk/commit/e2c7753bc9d80196d1c3e757efb627193d80f1eb